From stable-bounces@linux.kernel.org  Tue May 15 01:41:55 2007
Date: Tue, 15 May 2007 01:40:09 -0700 (PDT)
Message-Id: <20070515.014009.26532099.davem@davemloft.net>
To: stable@kernel.org
From: David Miller <davem@davemloft.net>
Cc: bunk@stusta.de
Subject: SERIAL SUNHV: Add an ID string.

The 'compatible' property can be SUNW,sun4v-console as
well as 'qcn'.

Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>

---
 drivers/serial/sunhv.c |    4 ++++
 1 file changed, 4 insertions(+)

--- linux-2.6.21.1.orig/drivers/serial/sunhv.c
+++ linux-2.6.21.1/drivers/serial/sunhv.c
@@ -493,6 +493,10 @@ static struct of_device_id hv_match[] = 
 		.name = "console",
 		.compatible = "qcn",
 	},
+	{
+		.name = "console",
+		.compatible = "SUNW,sun4v-console",
+	},
 	{},
 };
 MODULE_DEVICE_TABLE(of, hv_match);
